src/WebJobs.Extensions.DurableTask/DurableTaskJobHostConfigurationExtensions.cs

```diff
@@ -45,6 +46,9 @@ public static IWebJobsBuilder AddDurableTask(this IWebJobsBuilder builder)
             serviceCollection.TryAddSingleton<IMessageSerializerSettingsFactory, MessageSerializerSettingsFactory>();
             serviceCollection.TryAddSingleton<IErrorSerializerSettingsFactory, ErrorSerializerSettingsFactory>();
             serviceCollection.TryAddSingleton<IApplicationLifetimeWrapper, HostLifecycleService>();
+#if !FUNCTIONS_V1
+            serviceCollection.AddSingleton<ITelemetryActivator, TelemetryActivator>();
```

Comment:
We don't need to ever let customers inject their own custom implementation of this interface, right?

Reply:
@TsuyoshiUshio , could you please review connor's question above ^.

As for my own guess, think the answer is *no*. I don't see the use-case.

Reply:
No.  We don't need to let customer inject their won custom implementation. 
Some customer want open telemetry however, in this case, it should be done by PR, since there are other strategy classes are there e.g. `W3CTraceContext ...`  and `HttpCorrelationProtocol...`

Reply:
Fair enough. Sounds to me then like there's merit in leaving the possibility for customers to provide their customer implementations then.

Reply:
Let's leave it as `AddSingleton` for now, and if customers need to provide their own implementation later, we can open that up by converting this to `TryAddSingleton`